client/src/components/page-components/TopicSection.tsx

```diff
@@ -9,25 +9,29 @@ type Props = {
 };
 
 const TopicSection = ({ topic, color, stories }: Props) => {
-  return (
-    <>
-      <h2 className="pt-10 flex" style={{ color: color }}>
-        <Link to={`/${topic}`}>{topic?.toUpperCase()}</Link>
-      </h2>
-      <hr style={{ background: color }} className="h-[1px] border-0"></hr>
-      <div className="md:flex justify-between pt-5 md:pt-10">
-        {stories?.map((story) => (
-          <StoryCard
-            key={story.id}
-            id={story.id}
-            image={story.cover_url}
-            title={story.name}
-            body={story.body}
-          />
-        ))}
-      </div>
-    </>
-  );
```

Comment:
Nice, this makes the summary page look a lot more complete.

Reply:
Ooo, looks better now. I like that